draft-tschofenig-rats-psa-token.md

```diff
@@ -400,23 +415,71 @@ verifier may choose to ignore this claim in favor of other information.
+the intended semantics to the rest of the claims found in the token.
+
+The EAT `profile` (claim key 18) is used.  The following constraints
+apply to its type:
 
-The Profile Definition claim contains the name of a document that describes the
-"profile" of the report. The document name may include versioning. The value
-for this specification MUST be PSA_IOT_PROFILE_1.
+* The URI encoding MUST be used;
+* The value MUST be `http://arm.com/psa/2`.
+
+This claim MUST be present in a PSA attestation token.
+
+See {{sec-backwards-compat}}, for considerations about backwards compatibility
+with previous versions of the PSA attestation token format.
 
 ~~~
 {::include cddl/psa-profile.cddl}
 ~~~
 
+# Backwards Compatibility Considerations
+{: #sec-backwards-compat}
+
+Previous versions of this specification used different keys for the following
+claims:
+
+* Nonce (claim key -75008);
+* Instance ID (claim key -75009);
+* Profile Description (claim key -75000 and value `PSA_IOT_PROFILE_1`).
+
+These claim keys have been retired and their use is deprecated.
+
+Unless compatibility with existing infrastructure is a concern, emitters (e.g.,
+devices that implement the PSA attestation API) SHOULD produce tokens with
+their standard equivalent instead, as described in {{sec-nonce-claim}},
+{{sec-instance-id-claim}} and {{sec-profile-definition-claim}} respectively.
+
+In order to smooth the transition to the token format described in this
+document it is RECOMMENDED that receivers (e.g., PSA Attestation Verifiers)
+accept tokens encoded according to the old profile (`PROFILE_IOT_1`) as well as
+to the new profile (`http://arm.com/psa/2`), at least for the time needed to
+their clients to upgrade.
+
 # Token Encoding and Signing
 
-The report is encoded as a COSE Web Token (CWT) {{!RFC8392}}, similar to the
-Entity Attestation Token (EAT) {{?I-D.ietf-rats-eat}}. The token consists of a
-series of claims declaring evidence as to the nature of the instance of
-hardware and software. The claims are encoded in CBOR {{!RFC7049}} format.  For
-asymmetric key algorithms, the signature structure MUST be COSE_Sign1.  For
-symmetric key algorithms, the structure MUST be COSE_Mac0.
+The PSA attestation token is encoded in CBOR {{!RFC7049}} format.  Only
+definite-length string, arrays, and maps are allowed.
+
+Cryptographic protection is obtained by wrapping the `psa-token` map in a COSE
+Web Token (CWT) {{!RFC8392}}.  For asymmetric key algorithms, the signature
+structure MUST be COSE_Sign1.  For symmetric key algorithms, the signature
+structure MUST be COSE_Mac0.
+
+The CWT CBOR tag (61) is not used.  An application that needs to exchange PSA
+attestation tokens can use the Media Type defined in {{sec-iana-media-types}}
+or the CoAP Content-Format defined in {{sec-iana-coap-content-format}}.
+
+# Freshness Model
+
+The PSA Token supports the freshness models for attestation Evidence based on
```

Comment:
Should evidence be capitalised like this?

Reply:
It's the term of art established in the [RATS architecture document](https://ietf-rats-wg.github.io/architecture/draft-ietf-rats-architecture.html#section-8.1)